src/Core/Checkout/Payment/SalesChannel/PaymentMethodRoute.php

```diff
@@ -62,9 +61,7 @@ public function load(Request $request, SalesChannelContext $context, Criteria $c
 
         $paymentMethods = $result->getEntities();
 
-        if (Feature::isActive('cache_rework')) {
-            $paymentMethods->sortPaymentMethodsByPreference($context);
-        }
+        $paymentMethods->sortPaymentMethodsByPreference($context);
 
         /**
          * @deprecated tag:v6.7.0 - onlyAvailable flag will be removed, use Shopware\Core\Checkout\Gateway\SalesChannel\CheckoutGatewayRoute instead
```

Comment:
Is there a reason not directly removing this deprecation when you are already handling that class?

Reply:
because I was not sure, what to do here. and @cyl3x already took care here https://github.com/shopware/shopware/pull/6559